web-marketplace/src/lib/db/types/json-ld/project-metadata.ts

```diff
@@ -60,8 +60,30 @@ export interface ProjectPageMetadataLD {
   'regen:projectQuote'?: ProjectQuote;
   'regen:boundaries'?: string;
   'regen:landManagementActions'?: NameImageDescription[];
+
+  // Terrasos-specific fields
```

Comment:
I would assume these to be on chain if the project is on chain rather than in the project page metadata

Reply:
see suggestion: https://github.com/regen-network/regen-web/commit/111a15b9abda1ed22f2944340d4a153f970f6a7b

Reply:
I forget our motivation, but I know we decided to put all the data into project page metadata, rather than putting data into anchored metadata. @S4mmyb was this just to let us release sooner?

Reply:
> see suggestion: [111a15b](https://github.com/regen-network/regen-web/commit/111a15b9abda1ed22f2944340d4a153f970f6a7b)

so should I wait for this PR to be merged then and use it instead?

Reply:
I wanted to keep things consistent with how we’ve organized things so far: if the project is on-chain, store data that isn't purely for display purposes on-chain; if the project is off-chain, simply use the off-chain project metadata.

We only have one on chain terrasos project for now so this shouldn't be too much of an effort to put the data there, although it should work if in the interest of time, we keep it off chain for now... and put it on chain later.

Reply:
> > see suggestion: [111a15b](https://github.com/regen-network/regen-web/commit/111a15b9abda1ed22f2944340d4a153f970f6a7b)
> 
> so should I wait for this PR to be merged then and use it instead?

this is now merged into dev